package.json

```diff
@@ -23,11 +23,12 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "slugify": "^1.6.5",
+    "web3": "^1.8.2",
```

Comment:
Any reason to use `web3` instead of `ethers`? We already use `ethers` in this repo and across all of the other repos.